### PR TITLE
snap-telemetry 0.19.0

### PR DESCRIPTION
**Pull request by nanliu**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/snap-telemetry.rb [hunk 1]
```diff
@@ -22,37 +22,37 @@ def install
...
+    snapteld = buildpath/"src/github.com/intelsdi-x/snap"
+    snapteld.install buildpath.children
 
-    cd snapd do
+    cd snapteld do
       system "glide", "install"
-      system "go", "build", "-o", "snapd", "-ldflags", "-w -X main.gitversion=#{version}"
-      bin.install "snapd"
+      system "go", "build", "-o", "snapteld", "-ldflags", "-w -X main.gitversion=#{version}"
+      sbin.install "snapteld"
```
Comment (nijikon) [inline Formula/snap-telemetry.rb]: Why it was changed from `snapd` to `snapteld`?

File: Formula/snap-telemetry.rb [hunk 2]
```diff
@@ -22,37 +22,37 @@ def install
...
     end
 
-    snapctl = buildpath/"src/github.com/intelsdi-x/snap/cmd/snapctl"
-    cd snapctl do
-      system "go", "build", "-o", "snapctl", "-ldflags", "-w -X main.gitversion=#{version}"
-      bin.install "snapctl"
+    snaptel = buildpath/"src/github.com/intelsdi-x/snap/cmd/snaptel"
+    cd snaptel do
+      system "go", "build", "-o", "snaptel", "-ldflags", "-w -X main.gitversion=#{version}"
+      bin.install "snaptel"
```
Comment (nijikon) [inline Formula/snap-telemetry.rb]: Why it was changed from `snapctl` to `snaptel`?

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

File: Formula/snap-telemetry.rb (hunk 2 above)
Comment (MikeMcQuaid) [inline Formula/snap-telemetry.rb]: I think it's fine given the names they use in the tarball.

**Comment by nanliu** [comment]
In case anyone sees this issue and wonder about the binary name change. It's updated due to conflicts with Ubuntu snap project. For more info see: https://github.com/intelsdi-x/snap/issues/1344

**Pull request closed without merging**